src/react/components/pages/predict/predictPage.tsx

```diff
@@ -853,6 +887,8 @@ export default class PredictPage extends React.Component<IPredictPageProps, IPre
             ajax.then((response) => {
                 if (response.data.status.toLowerCase() === constants.statusCodeSucceeded) {
                     resolve(response.data);
+                    // prediction response from API
+                    console.log("raw data", JSON.parse(response.request.response));
```

Comment:
remove all console if check into master

Reply:
This is not my change, I just refactor the tslint error here. But I will remove it, if you think we don't need it here.